tests/test_helpers.py

```diff
@@ -27,8 +27,8 @@
      "2006as_timbuktoo has been added in expenses\n"),
     (["helper", "a_presentation", "Mars", "flat earth", "--type", "contributed_oral",
       "--begin_date", "2020-06-26", "--end_date", "2020-06-26", "--person", "ashaaban",
-      "--authors", "sbillinge", "--abstract", "the earth is round as seen from mars",
-      "--title", "On the roundness of the Earth"],
+      "--authors", "sbillinge", "ashaaban", "--abstract", "the earth is round as seen from mars",
+      "--title", "On the roundness of the Earth", "--status", "pending"],
```

Comment:
is pending one of the allowed values?

Reply:
I am not sure what the allowed values for status are, I am currently allowing any string to be a status.

Reply:
the allowed stati should be listed in the schema